lib/players/human.rb

```diff
@@ -0,0 +1,8 @@
+class Player::Human < Player
+  def move(board)
+    puts "#{self.token}, please choose a number between 1-9 to make your move."
+    player_move = gets.strip
+    player_move
```

Comment:
You don't need to store your return value in a local variable; just utilize the implicit return value of the expression.
```
def move(board)
  puts "#{self.token}, please choose a number between 1-9 to make your move."
  gets.strip
end
```